### ISSUE TITLE
API Request 

### ISSUE THREAD

**Issue by hooptie45** [magefree/mage#4215]
I'd really like to build a client driven by graphql subscriptions. I'd like to formalize the request/response data; and the game-state. 

It's hard to follow the client code as there is alot of UI code cluttering things up. I'm not looking to build a full UI, I just want to be able to drive a game using standard graphql. 

So if I was going to build something like this, where should I run it?

**Comment by JayDi85** [comment]
Client-server communications uses JBoss protocol. It's old and must be replace. Look at [that](https://www.slightlymagic.net/forum/viewtopic.php?f=116&t=18593) forum's post.
